<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>761</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_filepath">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>20</y>
      <width>571</width>
      <height>24</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Add_from_file_button">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>20</y>
      <width>41</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="chooseFileButton">
    <property name="geometry">
     <rect>
      <x>651</x>
      <y>20</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Save_Button">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>20</y>
      <width>41</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Alphabetical_sort_Button">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>460</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Model</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ID_sort_Button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Sort_by">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort by:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Add_from_search">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>41</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Search_Mode_Toggle">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>510</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ID">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>540</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_ID">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Model">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>540</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Model</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Model">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>540</y>
      <width>241</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_Type">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>540</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>EngineType</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Types">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>540</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>undefined</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>petrol</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>diesel</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>hybrid</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>electro</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_Horsepower">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>540</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>HP</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Horsepower">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>540</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_Occupied">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>570</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ID occupied</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Type_error">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>590</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>NOTE: QSpinBox for HP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Model_error">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>570</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Model can't be blank</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Horsepower_error">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>570</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Wrong input, HP &gt; 0</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>460</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Descending Order</string>
    </property>
   </widget>
   <widget class="QPushButton" name="horsepower_sort">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>460</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Horsepower</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Horsepower_search">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>520</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Approxiamtely (+- 10)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>877</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
